maths/3n+1.py

```diff
@@ -1,11 +1,17 @@
-def n31(a):
+from typing import Tuple, List
+
+def n31(a: int) -> Tuple[List[int], int]: 
     """
-    Returns Collatz sequence of number a
-    
+    Returns Collatz sequence of a number 
```

Comment:
This comment says that we are only returning the sequence but we are returning the sequence and its length.

Reply:
updated docstring